WordPress/Classes/ViewRelated/Blog/BlogDetailsViewController.m

```diff
@@ -137,10 +182,23 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
 
 - (void)configureCell:(UITableViewCell *)cell atIndexPath:(NSIndexPath *)indexPath
 {
-    if (indexPath.section == TableViewSectionContentType) {
+    if (indexPath.section == TableViewSectionGeneralType) {
+        switch (indexPath.row) {
+            case BlogDetailsRowViewSite:
+                cell.textLabel.text = NSLocalizedString(@"View Site", nil);
+                cell.imageView.image = [UIImage imageNamed:@"icon-menu-viewsite"];
+                break;
+            case BlogDetailsRowStats:
+                cell.textLabel.text = NSLocalizedString(@"Stats", nil);
+                cell.imageView.image = [UIImage imageNamed:@"icon-menu-stats"];
+                break;
+            default:
+                break;
+        }
+    } else if (indexPath.section == TableViewSectionPublishType) {
         switch (indexPath.row) {
-            case BlogDetailsRowPosts:
-                cell.textLabel.text = NSLocalizedString(@"Posts", nil);
+            case BlogDetailsRowBlogPosts:
+                cell.textLabel.text = NSLocalizedString(@"Blog Posts", nil);
```

Comment:
We've been actively removing any occurrence of the term "blog" from the app and using "site" instead.  However in this case I'm not clear why we'd not just say Posts.  Rationale for the change?